src/traces/scatter/attributes.js

```diff
@@ -65,6 +65,10 @@ module.exports = {
             'See `y0` for more info.'
         ].join(' ')
     },
+    identifier: {
```

Comment:
This one should be plural as it only make sense as an array. I'd vote for `identifiers` cc @cldougl .